Makefile

```diff
@@ -32,17 +32,17 @@ TEST_E2E_ARGS ?= '--ginkgo.focus=Port\sforwarding'
 
 ALL_ARCH.linux = amd64 arm arm64
 # as windows server core does not support arm64 windows image, trakced by the following link,
-# and only 1809 has arm64 nanoserver support, we support here only amd64 windows image
+# and only 1809/ltsc2019 has arm64 nanoserver support, we support here only amd64 windows image
 # https://github.com/microsoft/Windows-Containers/issues/195
 ALL_ARCH.windows = amd64
-ALL_OSVERSIONS.windows := 1809 2004 20H2 ltsc2022
+ALL_OSVERSIONS.windows := ltsc2019 ltsc2022
```

Comment:
@AbelHu could you help review this? thanks.

Reply:
I found below image is used in AKS Windows 2019 node. Does it still work with this change? Is the only difference that the image tag of nanoserver will be changed to ltsc2019?

mcr.microsoft.com/windows/nanoserver                             1809                          00a00b91628a0       103MB

Reply:
Yes the only difference is the tag. The digests and manifests have the same SHA

```bash
 marosset@marosset-z240:~$ regctl manifest get mcr.microsoft.com/windows/nanoserver:1809
Name:        mcr.microsoft.com/windows/nanoserver:1809
MediaType:   application/vnd.docker.distribution.manifest.list.v2+json
Digest:      sha256:adac2bb090398b9e2a9717571ee581816029424af706248b1cf3daae6178fc1f

Manifests:

  Name:      mcr.microsoft.com/windows/nanoserver:1809@sha256:534e38583dfa4ab78b76d5b8dd31da8db589d981cf1c0eb3c7b34fd6b69b1bbe
  Digest:    sha256:534e38583dfa4ab78b76d5b8dd31da8db589d981cf1c0eb3c7b34fd6b69b1bbe
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  windows/amd64/10.0.17763.4131
  OSVersion: 10.0.17763.4131
marosset@marosset-z240:~$ regctl manifest get mcr.microsoft.com/windows/nanoserver:ltsc2019
Name:        mcr.microsoft.com/windows/nanoserver:ltsc2019
MediaType:   application/vnd.docker.distribution.manifest.list.v2+json
Digest:      sha256:adac2bb090398b9e2a9717571ee581816029424af706248b1cf3daae6178fc1f

Manifests:

  Name:      mcr.microsoft.com/windows/nanoserver:ltsc2019@sha256:534e38583dfa4ab78b76d5b8dd31da8db589d981cf1c0eb3c7b34fd6b69b1bbe
  Digest:    sha256:534e38583dfa4ab78b76d5b8dd31da8db589d981cf1c0eb3c7b34fd6b69b1bbe
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  windows/amd64/10.0.17763.4131
  OSVersion: 10.0.17763.4131
  ```